src/org/thoughtcrime/securesms/ConversationAdapter.java

```diff
@@ -417,6 +432,7 @@ public void onBindLastSeenViewHolder(HeaderViewHolder viewHolder, int position)
 
     @Override
     protected boolean hasHeader(RecyclerView parent, StickyHeaderAdapter stickyAdapter, int position) {
+
```

Comment:
random extra line?

Reply:
was going to add code there but then forgot to remove it.